src/main/kotlin/sc/gui/controller/GameController.kt

```diff
@@ -198,6 +198,10 @@ class GameController : Controller() {
         }
         subscribe<HumanMoveRequest> { event ->
             logger.debug("Human move request")
+
+            val moves = GameRuleLogic.getPossibleMoves(event.gameState)
+            logger.debug("total: ${moves.size}")
```

Comment:
Remove the debug output or make the message more verbose so that it will be useful in the future.

Reply:
ah, yeah, my bad.
I used to print the whole list before, but removed that because it was too noisy.
I'll update it; it's useful for debugging